### DISCUSSION TITLE
IntelliJ IDEA Dark Themes

### DISCUSSION THREAD

**Discussion by davidsoles** [checkstyle/checkstyle#9328] [Ideas]
I found the checkstyle plugin for IntelliJ IDEA very useful but the it doesn't seen to play very well with dark themes. Each line background is gray. Here is an example.

![Screen Shot 2021-02-24 at 9 43 38 PM](https://user-images.githubusercontent.com/984410/109090458-5d515180-76e9-11eb-977f-7254144c5b00.jpg)

But in the problems tab looks fantastic.

![Screen Shot 2021-02-24 at 9 41 45 PM](https://user-images.githubusercontent.com/984410/109090268-195e4c80-76e9-11eb-88e3-450c3134ccb7.jpg)

Thanks for the plugin.

**Comment by rnveach** [comment]
We are not the maintainers of the IntelliJ plug if that is the purpose of this issue. We maintain the main checkstyle library that the plugin connects to.

**Reply by davidsoles** [reply]
I didn't open an issue, just making an observation. Thanks for the reply.

**Reply by romani** [reply]
please share this feedback at https://github.com/jshiell/checkstyle-idea
